pulp/Containerfile

```diff
@@ -8,7 +8,7 @@ ARG PULP_DEB_VERSION=""
 ARG PULP_FILE_VERSION=""
 ARG PULP_MAVEN_VERSION=""
 ARG PULP_PYTHON_VERSION=""
-ARG PULP_RPM_VERSION="~=3.13.2"
+ARG PULP_RPM_VERSION="~=3.14.0"
```

Comment:
Should I get rid of this entirely?  Or should we fill out the required versions for all other plugins?

Reply:
I have had the same question earlier that day. For consistent builds, pinning all plugins to y (as done here) would be good.
This probably depends on whether we want to use those containers in integration tests.